packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -83,6 +102,19 @@ export function BlockTypesTab(
 	// Hide block preview on unmount.
 	useEffect( () => () => onHover( null ), [] );
 
+	const getRootBlockTitle = () => {
+		const blockName = getBlockName( rootClientId );
+		// Find the title within the list of all possible blocks.
+		const block = allItems.find( ( item ) => item.name === blockName );
```

Comment:
Are we getting the block title here? Why not check the block type instead of looping through all blocks, like we normally do?